README.md

```diff
@@ -1,24 +1,172 @@
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    
+  </ul>
+</details>
+<br>
 
-* How to run the test suite
+## Key Features <a name="key-features"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- Registered users will be able to create new blog posts.
 
-* Deployment instructions
+- Users will be able to view blog posts on the application's home page.
 
-* ...
+- Users will be able to delete their own blog posts.
+
+- Users will be able to comment on blog posts.
+
+## 🚀 live-demo <a name="live-demo"></a>
+
+Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby installed on your machine
+- Text editor (VSCode, Sublime, Atom, etc.)
+- Install Rails gem
+
+### SetUp
+
+Clone this Repository to your desired folder:
+
+```sh
+cd my-folder
+git clone git@github.com:Rachelwebdev/blog-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog-app
+```
+
+### Dependencies
+
+Install Dependencies with:
+
+```sh
+  bundle install
+```
+
+create database in your postgre with:
+
+```
+  rails db:create
+```
+
+Note: If you have any problem with the database, you can check the database.yml file and change the username and password to your own. Run migration with:
+
+```sh
+  rails db:migrate
+```
+
+### Usage
+
+Run Project with:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+Run tests with:
+
+```sh
+    rspec
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Rachel Isaac**
+
+- GitHub: [@rachelwebdev](https://github.com/Rachelwebdev)
+- Twitter:[@rachelisaac13](https://twitter.com/Rachelisaac13)
+- LinkedIn: [Rachel Isaac](https://www.linkedin.com/in/rachelisaac13/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add Styling to the Project
+- Add Authentication page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like the project, give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+[Microverse](https://www.microverse.org/) for the oppotunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
**You did a great job! Your migration files look stunning!**

- [ ] However, your MIT license looks broken. Kindly add the correct links to your MIT license. Thank you!

Reply:
[OPTIONAL] I can see you ignored node_modules. But it is still on your GitHub. Having, node_modules in your GitHub is going to slow your PR and make it almost impossible to review. To avoid such issues kindly delete node_modules from your GitHub. Thank you!